docs/products/cassandra/howto/zdm-proxy.rst

```diff
@@ -0,0 +1,255 @@
+Migrate to Aiven for Apache Cassandra® with no downtime
+=======================================================
+
+Zero Downtime Migration (ZDM) Proxy is an open-source component developed in Go and based on client-server architecture. It enables you to migrate from one Apache Cassandra® cluster to another without downtime or code changes in the application client.
+
+.. seealso::
+
+   For details on ZDM Proxy, check out `zdm-proxy GitHub <https://github.com/datastax/zdm-proxy>`_.
+
+This article shows how to use ZDM Proxy to migrate your Apache Cassandra instances to Aiven.
+
+How it works
+------------
+
+When using ZDM Proxy, the client connects to the proxy rather than to the source cluster. The proxy connects both to the source cluster and the target cluster. It sends read requests to the source cluster only, while write requests are forwarded to both clusters.
+
+.. seealso::
+
+   For details on how ZDM Proxy works, check out `Introduction to Zero Downtime Migration <https://docs.datastax.com/en/astra-serverless/docs/migrate/introduction.html>`_.
+
+Prerequisites
+-------------
+
+* Apache Cassandra instance that you want to migrate to the Aiven platform (migration source)
+* Aiven for Apache Cassandra service where you want to migrate your external instance (migration target)
+* :doc:`Aiven CLI client installed </docs/tools/cli>`
+* ``cqlsh`` `installed <https://cassandra.apache.org/doc/latest/cassandra/getting_started/installing.html>`_
+
+Migrate to Aiven
+----------------
+
+Connect to the target
+'''''''''''''''''''''
+
+:doc:`Connect to your Aiven for Apache Cassandra service </docs/products/cassandra/howto/connect-cqlsh-cli>` using ``cqlsh``, for example.
+
+.. code-block:: bash
+
+    cqlsh --ssl -u avnadmin -p YOUR_SECRET_PASSWORD cassandra-target-cluster-name.a.avns.net 12345
+
+You can expect to receive output similar to the following:
+
+.. code-block:: bash
+
+    Connected to a1b2c3d4-1a2b-3c4d-5e6f-a1b2c3d4e5f6 at cassandra-target-cluster-name.a.avns.net:12345
+    [cqlsh 6.1.0 | Cassandra 4.0.11 | CQL spec 3.4.5 | Native protocol v5]
+
+Create keyspaces and tables
+'''''''''''''''''''''''''''
+
+In your target service, create the same keyspaces and tables you have in your source Apache Cassandra cluster. For ``replication_factor``, specify the number of nodes that the target cluster has.
```

Comment:
I think we could omit this (in general replication factor 3 might be a reasonable choice for a cluster of any size)
(I mentioned this in the draft just to make sure that it's not forgotten - it's easy to blindly copy DDLs from source to target)